### ISSUE TITLE
Remove only delete info on index but fetch always show them

### ISSUE THREAD

**Issue by sebglon** [hayorov/helm-gcs#6]
I deploy my chart
```
helm gcs push my-chart-0.0.1.tgz my-repo
``` 

After that i fetch it
```
helm fetch my-repo/my-chart --version 0.0.1 
```

All is good.

But now i want to delete it;
 ```
helm gcs remove my-repo/my-chart 
```
On my index, it is removed but tgz file always exist.

And fetch always see it.

is it possible to fetch chart only if it is present on index?
And add option on remove to force delete tgz?

**Comment by nouney** [comment]
Hi @sebglon,

I can't reproduce this issue. Does it happen every time for you?
Could you run `helm gcs remove my-repo/my-chart -d` please ? The `-d` flag is available since 0.1.3, you may need to update the plugin:

```shell
$ helm plugin gcs update
$ helm gcs version
version: 0.1.4
commit: 519b95ca3d0bde2ac1654ecb6b4995663b2a2eb5
date: 2018-03-20T22:47:57Z
```

**Comment by sebglon** [comment]
Hi @nouney ,

I always have an error with fetch cmd
I want to fetch latest release.
If i list all chart version available, i can see  version `1.0.0`

```
$ helm search akeneo-charts/onboarder -l
NAME                   	CHART VERSION  	APP VERSION	DESCRIPTION             
akeneo-charts/onboarder	1.0.0          	1.0.0      	Akeneo Onboarder package
akeneo-charts/onboarder	1.0.0-PR-7     	           	Akeneo Onboarder package
akeneo-charts/onboarder	0.0.9          	           	Akeneo Onboarder package
akeneo-charts/onboarder	0.0.4-CLOUD-594	0.0.22     	Akeneo Onboarder package
akeneo-charts/onboarder	0.0.3-CLOUD-594	0.0.22     	Akeneo Onboarder package
akeneo-charts/onboarder	0.0.2-CLOUD-594	0.0.22     	Akeneo Onboarder package
akeneo-charts/onboarder	0.0.1          	0.0.24     	Akeneo Onboarder package
```

But when i want to fetch latest chart release, i have an error if i not set `--version`
```
$ helm fetch akeneo-charts/onboarder --debug
Error: storage: object doesn't exist
Usage:
  helm-gcs pull gs://bucket/path [flags]

Flags:
  -h, --help   help for pull

Global Flags:
      --debug                    activate debug
      --service-account string   service account to use for GCS

Error: plugin "pull.sh" exited with error
```
